src/components/EnterpriseApp/index.jsx

```diff
@@ -10,6 +10,7 @@ import RequestCodesPage from '../../containers/RequestCodesPage';
 import Sidebar from '../../containers/Sidebar';
 import ReportingConfig from '../../components/ReportingConfig';
 import SupportPage from '../../containers/SupportPage';
+import SamlConfiguration from '../../containers/SamlConfiguration';
```

Comment:
Would Recommend: `SAMLConfiguration` instead

Reply:
That's generally inconsistent with Pascal's case which is used for JS class names (ie the ClassName capitalization), which recommends only using capitalized acronyms for 2 letter acronyms like "IO" though I see whoever worked on ReportingConfiguration disregarded it as well (I see a class "SFTPDeliveryMethodForm").
I know the backend is `SAMLProviderConfig`, but this isn't really meant to match, if it was JS would be written in snake case lol.

Reply:
I should clarify: it's fine to change it to match the formatting on the backend, it's just not consistent with the naming scheme we're using here (nor is the backend name really)

Reply:
I see, I guess I have not dug much into how js naming wants acronyms to be handled, but based on your explanation this seems apt.  I would not chagne anything here then, I just read this interesting thread https://github.com/airbnb/javascript/issues/653 did not quite get what their conclusion was lol

Reply:
ok wait I did find this one, https://github.com/airbnb/javascript#naming--Acronyms-and-Initialisms but again don't need to change anything, this maybe worth a check in fedx at some point I am curious too now